.github/workflows/build_deploy.yml

```diff
@@ -1,17 +1,21 @@
 name: Publish Docker image and deploy it
 
 on:
+  release:
+    types: [published]
   push:
     branches:
-      - develop
       - main
```

Comment:
I think it may be worth taking off `main` if we are going to scrap it at some point.